sections/futures/Trade/TradePanelHeader.tsx

```diff
@@ -26,10 +30,15 @@ export default function TradePanelHeader({ accountType, onManageBalance }: Props
 
 	const dispatch = useAppDispatch();
 	const theme = useTheme();
-	const wallet = useAppSelector(selectWallet);
 	const { openConnectModal } = useConnectModal();
 	const position = useAppSelector(selectPosition);
-	const balance = position ? position.remainingMargin : null;
+	const futuresType = useAppSelector(selectFuturesType);
+	const balanceInfo = useAppSelector(selectCrossMarginBalanceInfo);
+	const wallet = useAppSelector(selectWallet);
+
+	const balance = position ? position.remainingMargin : zeroBN;
```

Comment:
Minor change, but maybe we can do this instead?
```suggestion
	const balance = position?.remainingMargin ?? zeroBN;
```